src/napari_imagej/setup_imagej.py

```diff
@@ -39,17 +39,17 @@ def imagej_init():
     )
     config.add_option(f"-Dimagej2.dir={os.getcwd()}")  # TEMP
     config.endpoints.append("io.scif:scifio:0.43.1")
+    config.endpoints.append("org.scijava:scijava-common:2.89.0")  # TEMP
     log_debug("Completed JVM Configuration")
 
     # Parse imagej settings
     ij_dir = setting("imagej_installation")
 
-    _ij = imagej.init(ij_dir_or_version_or_endpoint=ij_dir, mode=get_mode())
+    _ij = imagej.init(
+        ij_dir_or_version_or_endpoint=ij_dir, mode=get_mode(), add_legacy=False
```

Comment:
We could  just make this a setting. Do you think people would find that useful @ctrueden?

Reply:
With more thinking, I'd prefer we do it later. We can then gauge how much of a priority it is and make sure it is done correctly.

But I did clean up the settings in 6d7c169